content/posts/2020/02/libplanet-0.8.eng.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Libplanet 0.8 Released
+date: 2020-02-05
+authors: [chanhyuck.ko, hong.minhee]
+translators: [kidon.seo]
+---
+
+Hello All.
+
+Our team has released the eighth minor version of [Libplanet], [Version 0.8][0.8.0].
+
+Libplanet is a common library that solves game implementation problems such as P2P communication and data synchronization when creating online multiplayer games that run on distributed P2P.
+
+With the new version updated in nearly three months, there have been many improvements in version 0.8 including storage optimization. This article will cover major changes of Version 0.8.
+
+[Libplanet]: https://libplanet.io/
+[0.8.0]: https://github.com/planetarium/libplanet/releases/tag/0.8.0
+
+
+Genesis Block Anticipation 
+--------------------------
+
+`BlockChain<T>` now anticipates a particular genesis block. This is to prevent mistakes such as multiple games attempting to connect to the wrong network. Mistakes like above might occur because although games made from Libplanet forms separate networks, Libplanet behaves in a type of metaprotocol.
```

Comment:
```suggestion
`BlockChain<T>` now anticipates a particular genesis block. This is to prevent mistakes such as multiple games attempting to connect to the wrong network. Mistakes like above might occur because although games made from Libplanet form separate networks, Libplanet behaves in a type of metaprotocol.
```